modules/iam-read-only/README.md

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
why not  have generic module for creating group and moving/attaching users into it, and also have policy configuration passed as variable, and there is examples have example how to create read-only access having group

 like we have for users module:
https://github.com/dasmeta/terraform-aws-modules/tree/main/modules/aws-iam-user


and also rename module "iam-group" ?